frontend/src/Editor/QueryManager/QueryEditors/Restapi/index.jsx

```diff
@@ -17,6 +17,7 @@ class Restapi extends React.Component {
     );
     this.state = {
       options,
+      arrayValuesChanged: false,
```

Comment:
Is this state being used directly? Looks like we are only setting it within options to be used with.